webpack.config.js

```diff
@@ -114,17 +110,17 @@ module.exports = function (env) {
   var serviceWorkerConfig = {
     clientsClaim: true,
     importScripts: [
-      '/service-worker/' + env.applicationName + '/service-worker.js'
+      '/service-worker/languageforge/service-worker.js'
     ],
     include: [] // To be changed once ready to start caching the whole app
   };
 
   // Set up and return a customized Webpack config according to the environment we're in
 
-  webpackConfig.entry.main = './src/angular-app/' + env.applicationName + '/main' +
+  webpackConfig.entry.main = './src/angular-app/languageforge/main' +
     (env.isTest ? '.specs' : '') + '.ts';
 
-  if (env.isProduction) {
+  if (env.NODE_ENV == "production") {
```

Comment:
```suggestion
  if (process.env.NODE_ENV == "production") {
```

Reply:
Looks good.

Reply:
I'll let you commit this since if I commit it, you'll end up having to rebase your branch next time you try to push it to GitHub and that would not be worth the complexity of the rebase for such a simple change.